ci/generate-markdown/update-library-landing-dox.sh

```diff
@@ -206,6 +206,11 @@ This assumes you have created a custom idempotency policy. Such as:
 
 @snippet $(basename "${sample_cc}") custom-idempotency-policy
 
+If long running operations exist, this will override the polling
+policies for \`${client_name}\`
+
+@snippet $(basename "${sample_cc}") set-polling-policy
+
```

Comment:
I think we should have the generator leave some marker that we use to conditionally include this in the document.